src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { DropdownButton } from "react-bootstrap";
+import classNames from "classnames";
+
+import { ContainerClassName } from "types/utility";
+
+import "./Dropdown.scss";
+
+export interface DropdownProps extends ContainerClassName {
+  id: string;
```

Comment:
Why is the `id` prop added and mandatory on top of that?

Reply:
fixed in dc3dcc0

comment above https://github.com/sparkletown/sparkle/pull/2359#discussion_r718510885